src/components/TrendChartDetails.js

```diff
@@ -16,16 +16,10 @@ const getComparison = ({ place, data }) => {
   const placeRate = data[place].rate
   const nationalRate = data[nationalKey].rate
   const diff = (placeRate / nationalRate - 1) * 100
-  const diffAbs = Math.abs(diff)
 
-  return diffAbs < threshold
-    ? <span>{highlight(`about the same (within ${threshold}%)`)} as</span>
-    : <span>
-        {highlight(
-          `${formatRate(diffAbs)}% ${diff > threshold ? 'higher' : 'lower'}`,
-        )}
-        {' '}than
-      </span>
+  return Math.abs(diff) < threshold
+    ? <span>the same as</span>
+    : <span>{highlight(`${diff > 0 ? 'higher' : 'lower'}`)} than</span>
```

Comment:
added threshold back (for now) when rates are "about the same" -- let's open a new issue if we want to amend copy further